### ISSUE TITLE
Problem with keyboard step, fixed after reinstalling OpenJDK from setup guide 

### ISSUE THREAD

**Issue by filip86pa** [kelaberetiv/TagUI#1143]
Hello

I have problem with keyboard commands. Keyboards commands do not work. My windows is 7 professional, 
Please suggest me how to solve the problem.

Thank you

**Comment by ruthtxh** [comment]
Hi @filip86pa, may I confirm that you have installed java (openJDK) on your windows? If not, you can follow the installation guide here, step 3.

https://tagui.readthedocs.io/en/latest/setup.html

**Comment by filip86pa** [comment]
Thank you
I reinstalled, now is ok

**Comment by Shubhavatar** [comment]
I'm facing a similar issue:
The scripts work fine on my machine, but the keyboard commands do not work on my user's machine.
I've tried reinstalling OpenJDK (the amazon corretto - and also tried repairing the installation on their machine after reading this thread).
I also checked if the antivirus is causing any blocks on their machine, but that does not seem to be the case.
They are using a 64-bit windows system.
